Develop

Mitlosh opened the pull request:
https://mitlosh.github.io/layout_landing-page/

anastasiiavorobiova reviewed (changes requested):
Great start!
- <img width="136" alt="Screenshot 2025-05-07 at 19 15 20" src="https://github.com/user-attachments/assets/fb56799c-ed2d-446d-950b-bdd856407c02" />
- Consider showing the phone number on the icon hover
- <img width="539" alt="Screenshot 2025-05-07 at 19 15 46" src="https://github.com/user-attachments/assets/4c48998b-af4f-4451-8fde-7aad8f534c0d" />
- Check font style here
- <img width="1304" alt="Screenshot 2025-05-07 at 19 16 15" src="https://github.com/user-attachments/assets/2dd24430-ff04-4536-a5a0-78e167035414" />
- Check the button size for desktops
- <img width="549" alt="Screenshot 2025-05-07 at 19 16 21" src="https://github.com/user-attachments/assets/167e895d-86f5-45ac-b0fb-654a0fee866d" />
- Consider removing default outlines
- <img width="282" alt="Screenshot 2025-05-07 at 19 16 26" src="https://github.com/user-attachments/assets/46611b0c-d6db-4bf4-a3a6-bad7153a6583" />
- Consider using `tel`, `mailto` link, link to Google Maps
- <img width="631" alt="Screenshot 2025-05-07 at 19 16 35" src="https://github.com/user-attachments/assets/b6f3e2e0-b35b-49d9-b7b7-158dcc84b6ac" />
- A user shouldn't be able to resize the textarea
- <img width="764" alt="Screenshot 2025-05-07 at 19 16 59" src="https://github.com/user-attachments/assets/0efd03d9-8130-4ac2-910d-a2994b69742d" />
- The form leads to the 404 page after submission. Consider just clearing the form
- <img width="839" alt="Screenshot 2025-05-07 at 19 17 21" src="https://github.com/user-attachments/assets/87f820d1-e7b7-426f-a6af-cd62a5e7b772" />
- The page under the menu shouldn't be scrollable

Mitlosh commented:
- added phone number on icon hover 
- changed recommended products styles
- adjusted categories button width on desktop
- removed default input outline and added hover and focus effect like in figma design
- added tel, mailto link, link to Google Maps in footer and also in top-bar phone number
- disabled textarea resize and adjusted its height
- the form is cleared after submit. Used javascript
- page doesn't scroll when the burger menu is open

VitaliyBondarenko1982 reviewed (changes requested):
Looks like fixes wasn't deployed or not fixed.
Please check demo link.
- no visible phone number on hover in header.
- scroll on page still exist under opened menu
![Screenshot 2025-05-08 at 15 28 06](https://github.com/user-attachments/assets/b5c0d54e-0927-46b7-a3c1-49e6a61e5778)
check other points from previous review comment.

Mitlosh commented:
Sorry, forgot to deploy. Should work now

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
To improve:
1. Add the topical title to the page 
<img width="139" alt="image" src="https://github.com/user-attachments/assets/3fb6a254-45c0-4d55-b779-011446edfa68" />

2. I can't scroll page after clicking on these links 
<img width="383" alt="image" src="https://github.com/user-attachments/assets/d3613f22-1f44-4b39-8201-b453b6f83d54" />
<img width="147" alt="image" src="https://github.com/user-attachments/assets/aa678617-2658-4175-81c4-9ded01ea5247" />

3. The prices should be are on the same horizontal line 
<img width="1226" alt="image" src="https://github.com/user-attachments/assets/efa878ef-b403-43ef-8d4a-33cf333d2031" />

4. The font size looks different as in the design 
<img width="453" alt="image" src="https://github.com/user-attachments/assets/8c9b2bea-055d-4720-9206-a904b9874da8" />

5. Make these fields required before submit 
<img width="497" alt="image" src="https://github.com/user-attachments/assets/539601a9-38e7-4ad2-a6ed-de587124c02f" />

etojeDenys reviewed (approved):
good job overall
